_includes/wiki/en/footertext.md

```diff
@@ -1,6 +1,6 @@
 
 
-**Summer 2020! Tell us about Jamulus - [take our anonymous survey!](https://forms.gle/hSSjsxjWj2Pnp5kr7)**
+**Tell us about Jamulus - [take our anonymous survey!](https://forms.gle/hSSjsxjWj2Pnp5kr7)**
```

Comment:
I thought you were doing something like a hook which automatically updated changes with modifications made to release?

Reply:
Any way, done: https://github.com/jamulussoftware/jamuluswebsite/commit/38511a38b4372e3843a2883dc2a0eb3f4e5e0cde

Reply:
Yes. But this doesn't include content which needs to be translated. So if you change something in french and merge it to release, it will also add the commit to changes. But obviously this isn't possible for multiple languages.

Reply:
You're right. Thanks for the explanation and sorry for the mistake here.